pkg/registry/core/service/allocator/utils_test.go

```diff
@@ -21,20 +21,6 @@ import (
 	"testing"
 )
 
-func TestBitCount(t *testing.T) {
```

Comment:
makes sense to still include a unit test that makes sure the function `countBits` gives the correct count.

Reply:
actually never mind, i've missed TestCountBits bellow.

/ok-to-test